docs/testing/integration_tests.rst

```diff
@@ -2,4 +2,50 @@
 Integration Tests
 *****************
 
-- To be completed
+
+Configuration
+=============
+
+The main configuration file to look at is ``/tests/protractor.conf.js``.
+It configures our **browserName**.
```

Comment:
Just spotted this - if "browserName" is a literal (e.g. a variable name) it should be in double-backticks - ``browserName``. See http://docs.django-cms.org/en/develop/contributing/contributing.html#inline-markup.